### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by SpringrollBrothers**
HI, this is my home work for JS WEEK1:
Age-ify
Goodboy-Oldboy 
Housey pricey 
Ez Namey

### CODE REVIEW

**Review by nishadipri — COMMENTED** [review]
Good Job 👍🏽 . You can also use template literal to write the output message. Try it next time. It will improve readability .

console.log(`You will be ${age} years old in ${yearOfFuture}.`);

File: javascript/javascript1/week1/script.JS [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const yearOfBirth = 1987;
+const yearOfFutrue=2027;
+const age = yearOfFutrue - yearOfBirth;
+console.log("You will be "+age+ " years old in "+yearOfFutrue+".");
+
+
+const dogYearOfBirth= 2021;
+const dogYearFuture=2027;
+let dogYear= dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears= true;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/script.JS:10]: Those could be set as `const` as you never reassign them.

File: javascript/javascript1/week1/script.JS [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    {
+        dogYear*=7;
+console.log("Your dog will be "+ dogYear+" dog year old in " +dogYearFuture+".");
+}
+else{
+    console.log("Your dog will be " +dogYear+" human years old in"+dogYearFuture+".");
+}
+
+
+const houseOfPeter=8*10*10;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/script.JS:21]: This variable name can be improved upon, as it's not clear what it really holds. In this case it's volume in square meters. Example:
```
const peterHouseM2 = 8*10*10;
```

File: javascript/javascript1/week1/script.JS [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+
+const randomNumber1 = Math.floor(Math.random() * 10);
+const randomNumber2 = Math.floor(Math.random() * 10);
+
+const firstWords=["fabulous","splendid","mysterious","magical","blur","bluffing","sweet","scary","jumping","sleeping"]
+const secondWords=["spaceship","robot","yoga","elephant","box","burger","vegie","giraffe","mammi","tomato"]
+let startupName =firstWords[randomNumber1]+" "+secondWords[randomNumber2]
+let startupNameLength= startupName.length
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/script.JS:49]: Those can be set as `const`

**Review by jason-vasilev — COMMENTED** [review]
As Nish has written, you could utilize some [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for cleaner messages. Also remember to always use `const` until it doesn't `let` you to.

A good practice you could adopt is to document your code through comments. I don't think we showed it in class, but most probably you've already seen it through FreeCodeCamp `// ` is for a single line comment and `/* some text here */ ` is for multiple lines.

**Pull request merged**